tools/code_format/check_format.py

```diff
@@ -905,6 +906,14 @@ def clangFormat(file_path):
     return ["clang-format rewrite error: %s" % (file_path)]
   return []
 
+# Fix for https://github.com/envoyproxy/envoy/issues/10535
+def checkProtoiValidation(file_path):
```

Comment:
What does "Protoi" mean? Should it just be "Proto"?

Reply:
yes, it is typo. i will correct it.